src/main.test.js

```diff
@@ -54,35 +54,35 @@ test("Valid input with comment,middle of line", () => {
 test("Valid input with comment, start of line", () => {
   const testEnv =
     "#DATABASE_URL: email = test@mail.com\nDATABASE_URL: email = test@mail.com";
-  expect(envSpecToHTML(testEnv)).toEqual(
+  expect(envSpec.parse(testEnv)).resolves.toEqual(
     `<label for="env_spec_database_url">DATABASE_URL</label>\n` +
       `<input id="env_spec_database_url" name="database_url" type="email" value="test@mail.com" />\n`
   );
 });
 
 test("Invalid environmental variable : starts with number", () => {
   const testEnv = "1DATABASE_URL\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpec.parse(testEnv)).rejects.toEqual("Error:Wrong Syntax");
 });
 
 test("Invalid environmental variable : contains non alphanumeric characters", () => {
   const testEnv = "DATABASE_URLαα\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpec.parse(testEnv)).rejects.toEqual("Error:Wrong Syntax");
 });
 
 test("Invalid environmental variable : contains lowercase letters", () => {
   const testEnv = "database_url\nADMIN_EMAIL:email";
-  expect(envSpecToHTML(testEnv)).toEqual("Error:Wrong Syntax");
+  expect(envSpec.parse(testEnv)).rejects.toEqual("Error:Wrong Syntax");
```

Comment:
You need to `return` this, in order to mark the test successfully as passed/failing ([see example](https://jestjs.io/docs/en/expect.html#rejects)).